<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PixivChecker</class>
 <widget class="QMainWindow" name="PixivChecker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pixiv图片查看</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>531</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>图片设置</string>
    </property>
    <widget class="QPushButton" name="PIDSubmit">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>110</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>491</width>
       <height>85</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelPID">
          <property name="text">
           <string>PID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editPID">
          <property name="whatsThis">
           <string/>
          </property>
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>如：82775556</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="isMultiPage">
            <property name="text">
             <string>多页插画</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="labelPage">
              <property name="text">
               <string>插画页数</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="editPage">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>9999</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="labelType">
              <property name="text">
               <string>图片后缀名</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="pngButton">
              <property name="tabletTracking">
               <bool>false</bool>
              </property>
              <property name="acceptDrops">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>png</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="jpgButton">
              <property name="text">
               <string>jpg</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="gifButton">
              <property name="text">
               <string>gif（动图）</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="isMirror">
            <property name="text">
             <string>使用镜像域名（推荐）</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>560</y>
      <width>371</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="SavePicture">
       <property name="text">
        <string>保存图片</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ReloadPicture">
       <property name="text">
        <string>刷新图片</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="About">
       <property name="text">
        <string>关于..</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="PictureView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>530</width>
      <height>375</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>530</width>
      <height>375</height>
     </size>
    </property>
    <property name="text">
     <string>图片将在这里展示...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
